### DISCUSSION TITLE
请问该如何给yutto添加UA？ 我自己尝试fork仓库并修改了两段代码，但我并不清楚是否生效。

### DISCUSSION THREAD

**Discussion by OcrSu** [yutto-dev/yutto#397] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/a896e131-bf97-47af-936d-6b0e3a23686e)
![image](https://github.com/user-attachments/assets/3773c776-8da2-44b3-a678-a1b0b7c00ea5)
修改的代码如上

我从这里查到的https://scrapingant.com/blog/change-user-agent-httpx

但我完全没学过任何编程语言，完全不懂这样做是否生效。

**Comment by SigureMo** [comment, marked as answer]
https://github.com/yutto-dev/yutto/blob/244c51ff798b1a244900df6f3706e202805ccf16/src/yutto/utils/fetcher.py#L62
